mod-orders-storage/schemas/piece.json

```diff
@@ -41,11 +41,20 @@
       "description": "UUID of the holding record",
       "$ref": "../../common/schemas/uuid.json"
     },
+    "receivingTenantId": {
+      "description": "receiving tenant for ECS-enabled clusters",
```

Comment:
all other descriptions in this file started with capital letter, so better to keep the same convention

Reply:
Made all descriptions capitalized